src/containers/simulator/simulatorStatus.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+import { bindActionCreators } from 'redux'
+import { connect } from 'react-redux'
+
+import './simulatorStatus.css';
+
+const SimulatorStatus = ({ isRunning, isInitialised, parseResults, roundResult }) => (
+  <section id="simulatorStatus">
+    <span className={isRunning ? `fade` : ``}>{getStatusMessage(isRunning, isInitialised, parseResults, roundResult)}</span>
+  </section>
+)
+
+const getStatusMessage = (isRunning, isInitialised, parseResults, roundResult) => {
+
+  if(parseResults.length === 0) {
+    return 'waiting for warriors'
+  }
+
+  if(roundResult.outcome) {
+    let outcome = `simulation complete - result(${roundResult.outcome})`;
+    if(roundResult.winnerId) {
+      outcome += ` - winner(${roundResult.winnerId})`
+    }
+    return outcome;
+  }
+
+  if(parseResults.length > 0 && !isInitialised && !isRunning) {
+    return `${parseResults.length} warriors loaded, awaiting init command`
+  }
+
+  if(parseResults.length > 0 && isInitialised && !isRunning) {
```

Comment:
Almost the same if condition twice in a row, could you combine them?

Reply:
Yeah, this bit is shit. non refactored rubbish just to see if is a sensible concept or not.

The logic is also floored, can you spot where?

Reply:
Floored or flawed?

Reply:
No, I can't :P